### PR TITLE
Platform events support

### PR DESCRIPTION
**Pull request by EmilioPeJu**
- Add platform event message
- Add function to send a platform event

### CODE REVIEW

File: pyipmi/msgs/message.py [hunk 1]
```diff
@@ -304,6 +304,11 @@ def __init__(self, name='picmg_identifier', value=None):
         UnsignedInt.__init__(self, name, 1, value)
 
 
+class EventMessageRevision(UnsignedInt):
+    def __init__(self, name='event_message_rev', value=None):
```
Comment (hthiery) [inline pyipmi/msgs/message.py:308]: Can the name have another value than 'event_message_rev'? Is it required that this is variable? If not we can drop that. And fill it fixed:

`UnsignedInt.__init__(self, 'event_message_rev', 1, value)`

File: pyipmi/msgs/sensor.py [hunk 2]
```diff
@@ -299,3 +300,27 @@ class GetSensorReadingRsp(Message):
...
             Optional(UnsignedInt('states2', 1)),
     )
+
+
+@register_message_class
+class PlatformEventReq(Message):
+    __cmdid__ = constants.CMDID_PLATFORM_EVENT
+    __netfn__ = constants.NETFN_SENSOR_EVENT
+    __fields__ = (
+        EventMessageRevision('event_message_rev', 4),
```
Comment (hthiery) [inline pyipmi/msgs/sensor.py:310]: If comment above is valid we can drop the first parameter.

File: pyipmi/msgs/sensor.py [hunk 3]
```diff
@@ -299,3 +300,27 @@ class GetSensorReadingRsp(Message):
...
+        UnsignedInt('sensor_number', 1),
+        Bitfield('event_type', 1,
+                 Bitfield.Bit('type', 7, 0),
+                 Bitfield.Bit('dir', 1, 0),),
+        RemainingBytes('event_data'),
+    )
+
+
+@register_message_class
+class PlatformEventRsp(Message):
```
Comment (hthiery) [inline pyipmi/msgs/sensor.py:321]: Please add a unit test for req and rsp.

File: pyipmi/msgs/sensor.py (hunk 2 above)
Comment (EmilioPeJu) [reply pyipmi/msgs/sensor.py]: Done

**Comment by EmilioPeJu** [comment]
Hi Heiko and thanks for reviewing this PR, I've done as suggested and also added an extra test for the platform event function

**Comment by hthiery** [comment]
Looks good. Thank you.

**Pull request merged**